Assets/_BForBoss/_Core/Scripts/Environment/HealingItemBehaviour.cs

```diff
@@ -6,6 +6,14 @@ namespace Perigon.Entities
     public class HealingItemBehaviour : ItemPickupBehaviour
     {
         [SerializeField] private float _healAmount = 50f;
+        private bool _shouldResetOnlyWhenWaveIncrements = false;
```

Comment:
Don't think you'll need an option to switch this. Since it's a global change, designers will probably touch this at most once or twice, and then never touch it again. 

Should probably setup the code in a way where you can revert back to timer style easily.